### PR TITLE
add a basic code climate config and corresponding badge to the README

### PR DESCRIPTION
**Pull request by cnelson**
As part of the Fedramp effort, we are required to ensure static code analysis is run on all 18F developed code that is part of the cloud.gov platform:  https://github.com/18F/cg-product/issues/260

In support of this effort, this PR adds a basic code climate configuration and badge to the README.

**Also whomever is reviewing this PR please provide me with access to this repo, so I can configure the webhook required for automatically running code climate on each PR**

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+engines:
+  duplication:
+    enabled: false
+  fixme:
+    enabled: true
+  radon:
+    enabled: true
```
Comment (afeld) [inline .codeclimate.yml:7]: Note that none of these are actually doing security scanning...

/cc https://github.com/18F/compliance-toolkit/issues/14

**Comment by afeld** [comment]
> whomever is reviewing this PR please provide me with access to this repo, so I can configure the webhook required for automatically running code climate on each PR

✅

**Comment by cnelson** [comment]
Yeah, there's not a great dedicated security scanner for python that I'm
aware of, but from a Fedramp perspective the critical thing is that we do
static analysis in general of our code (linting type checks count if
security scanners aren't available) in a standard way across all our
in-house developed code.  Based on the feedback I was given that should be
Code Climate and run at PR time.

Once we have this step in place for all repos in scope we can continue to
enhance it hopefully by contributing to the available engines in Code
Climate (for example, it might be worth investigating writing code climate
support for Mypy to at least get some static type checking)

For other code scanning related Fedramp controls, we are going to rely on
dynamic scanners that run on our hosts once this code makes it into our
test/staging environments.  This change in approach is based on emulating
what's happening in Fedramped lower layers of our ATO stack.

**Comment by afeld** [comment]
Maybe add an empty commit to ensure that the configuration/integration works? After that, feel free to merge!

**Pull request merged**